### ISSUE TITLE
Calendar Copy to Clipboard 

### ISSUE THREAD

**Issue by Odoamar** [fmasa/wfrp-master#252]
**Is your feature request related to a problem? Please describe.**
After every session or/and event in game, I try to prepare a quick summary. Copy paste feature will be the great addition.

**Describe the solution you'd like**
Simple button or pop up menu after long press on Calendar.

**Comment by fmasa** [comment]
Hi, do you mean that copy of a current date?

**Comment by Odoamar** [comment]
Yes current date plus all associated information ( Mannslieb phase, season )

**Comment by Odoamar** [comment]
yea sure
